base/compiler/ssair/inlining.jl

```diff
@@ -832,6 +833,13 @@ function analyze_method!(match::MethodMatch, argtypes::Vector{Any},
         # call this function
         return nothing
     end
+    if !match.fully_covers
```

Comment:
Even when `match.fully_covers`, `spec_types` can contain Varargs?
```julia
julia> @eval Core.Compiler let
           foo(a...) = length(a)
           matches = _methods_by_ftype(Tuple{typeof(foo),Int,Vararg{Int}}, -1, get_world_counter())
           matches[1]
       end
       
Core.MethodMatch(Tuple{Core.Compiler.var"#foo#431", Int64, Vararg{Int64}}, svec(), (::Core.Compiler.var"#foo#431")(a...) in Core.Compiler at none:2, true)
```

Reply:
If it fully covers, then we don't need inline the subtyping computation later (which is what failed)